egs/fame/s5/local/fame_dict_prep.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2015-2016  Sarah Flora Juan
```

Comment:
If you modified the file, add your (c),
if you are using it as it was, please add a note from what recipe this file was taken from